edxval/settings/base.py

```diff
@@ -180,6 +176,15 @@
 )
 
 
+STORAGES = {
```

Comment:
This is the default value in Django so I don't think you need to add it to our base settings file here.